src/ConfigurableFirmata.h

```diff
@@ -93,6 +93,11 @@
 #define SYSEX_NON_REALTIME      0x7E // MIDI Reserved for non-realtime messages
 #define SYSEX_REALTIME          0x7F // MIDI Reserved for realtime messages
 
+// DeviceFirmata commands
+
+#define DEVICE_QUERY            0x30 // message requesting action from a device driver (DeviceFirmata)
```

Comment:
I'd prefer if the command IDs for DEVICE_QUERY and DEVICE_RESPONSE were closer to the existing allocations, so perhaps 0x5E and 0x5F rather than 0x30 and 0x31. This is in case it's ever necessary to reserve a block of command IDs at some point in the future it keeps a wider range open.